platform/android/java/plugins/godotpayment/build.gradle

```diff
@@ -21,6 +21,7 @@ android {
 dependencies {
```

Comment:
With all the code deletion, seems you should also be able to remove the `useLibrary 'org.apache.http.legacy'`.